app/static/js/libs/marked.js

```diff
@@ -834,7 +834,7 @@ Renderer.prototype.listitem = function(text, checked) {
   }
 
   return '<li class="task-list-item">'
-    + '<input type="checkbox" class="task-list-item-checkbox"'
+    + '<input type="checkbox" class="task-list-item-checkbox" onchange = "checkUncheck(this)"'
```

Comment:
I think we can create a custom listitem renderer instead of overwriting the source code.